### DISCUSSION TITLE
How to embed a picture hosted on PhotoPrism?

### DISCUSSION THREAD

**Discussion by xplosionmind** [photoprism/photoprism#1898] [Q&A] [answered]
If I need to embed anywhere a picture on my Photoprism instance, how can I do so?

I just need the URL to put in the `img` `src` attribute.

Thank you very much!

**Comment by lastzero** [comment, marked as answer]
Currently you can right/long click on an image in the viewer and then select "Copy image address" to get the thumbnail URL, e.g. for the 1280px version:

https://youserver.com/api/v1/t/b825a27df67f6c1f1c329bc50e2e767d5b44a584/b482f5e1/fit_1280

More information on thumbnail types:

https://docs.photoprism.app/developer-guide/media/thumbnails/#thumbnail-types

Be aware that this exposes your URL security token (`b482f5e1` in the example above). This is not ideal for the highest possible security if you assume that an attacker could somehow find out the SHA1 file hash in the URL (e.g. if you have sensitive/illegal images in your library that others might also have, so they know the hash).

We will provide a general, more user-friendly and secure solution in a future release. There should be a few related GitHub Issues / Feature Requests you can vote for!
